src/native/external/llvm-libunwind-version.txt

```diff
@@ -3,3 +3,5 @@ https://github.com/llvm/llvm-project/releases/tag/llvmorg-14.0.6
 
 Apply https://github.com/dotnet/runtime/commit/e57194552327bccaf2b45a9190f0e6e1289472e4
 Apply https://github.com/dotnet/runtime/commit/db7ed089a9e20a391a53e0b547f95dc8bf916765
+Apply https://github.com/dotnet/runtime/commit/828c8d0b2a8cdc2479524c5229939641d3610a1b
```

Comment:
These are sha in your branch. They will change once this is merged.

Could you please submit these two llvm changes (without the change in llvm-libunwind-version.txt) in separate PR?

Reply:
> These are sha in your branch.

When a commit is pushed to an active PR, it is associated to the target GitHub repository. We can get any PR commit (attached or orphaned alike) with `https://github.com/dotnet/runtime/commit/{sha}.patch`.

If the contents of commit are not changing, this reference will remain valid.

Reply:
ok, I thought that we maintain these as links into the main branch history, but I see that it is not the case.